src/wazuh_modules/wm_sca.c

```diff
@@ -1830,7 +1830,17 @@ static int wm_sca_is_process(char *value, OSList *p_list)
 }
 
 // Destroy data
-void wm_sca_destroy(wm_sca_t * data) {
+void wm_sca_destroy(wm_sca_t *data) {
+
+    int i;
```

Comment:
The scope of the variable `i` should be inside the conditional if(data->profile)